Assets/LeapMotionModules/Interaction/Editor/InteractionBehaviourEditor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEditor;
+
+namespace Leap.Unity.UI.Interaction {
+
+  [CanEditMultipleObjects]
+  [CustomEditor(typeof(InteractionBehaviour), editorForChildClasses: true)]
+  public class InteractionBehaviourEditor : CustomEditorBase<InteractionBehaviour> {
+
+    private EnumEventTableEditor _tableEditor;
+
+    protected override void OnEnable() {
+      base.OnEnable();
+
+      deferProperty("_eventTable");
+      specifyCustomDrawer("_eventTable", drawEventTable);
+
+      specifyConditionalDrawing(() => !target.ignoreGrasping,
+                   "_allowMultiGrasp",
+                   "_moveObjectWhenGrasped",
+                   "graspedMovementType",
+                   "graspHoldWarpingEnabled__curIgnored");
```

Comment:
I would also like to draw attention to the drawing changes I made here, @nickjbenson 

These properties had [DisableIf] attributes on them before; I assume they were there because it was the farthest you could get without writing an editor script...